Microsoft.Toolkit.HighPerformance/Helpers/NullableHelper.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Runtime.CompilerServices;
+
+
+namespace Microsoft.Toolkit.HighPerformance.Helpers
+{
+    /// <summary>
+    /// Helpers to perform operations on nullable value types
+    /// </summary>
+    public static class NullableHelper
```

Comment:
This API is in the wrong place, it's an extension. It should be `NullableExtensions` and be in the `Microsoft.Toolkit.HighPerformance.Extensions` namespace. You can look at the others to see how they're structured.